File: Threading_Exercise_2/app/src/main/java/com/example/slapocolypse/threading_exercise/JokeService.java
package com.example.slapocolypse.threading_exercise;

import retrofit2.Call;
import retrofit2.http.GET;

/**
 * Created by slapocolypse on 3/12/18.
 */

public interface JokeService {
    @GET("jokes/random")
    Call<Joke> randomJoke();
}

File: Android 09 - Threading and Android, Coroutines/Threading_Exercise_2/app/src/main/java/com/example/slapocolypse/threading_exercise/JokeService.kt
package com.example.slapocolypse.threading_exercise

import retrofit2.Call
import retrofit2.http.GET

/**
 * Created by slapocolypse on 3/12/18.
 */
interface JokeService {
    @GET("jokes/random")
    fun randomJoke(): Call<Joke?>?
}